x-pack/plugins/spaces/public/views/nav_control/nav_control.tsx

```diff
@@ -54,7 +57,7 @@ module.controller(
     let mounted = false;
 
     $scope.$parent.$watch('isVisible', function isVisibleWatcher(isVisible: boolean) {
-      if (isVisible && !mounted) {
+      if (isVisible && !mounted && !pathProvider.isStatusPage()) {
```

Comment:
We'll want to replicate this check https://github.com/elastic/kibana/pull/24706/files#diff-ff2aa6f3a5836f032823714107a5b6a5L103

Reply:
I went back and forth on that originally -- we won't need it right away, because the K7 login page is opt-in, and the unauthenticated status page always renders with the default advanced settings, which uses the old nav bar. I can add it in now so it doesn't break for 7.0 though